python/filtering/filter_menu.py

```diff
@@ -275,7 +282,7 @@ def active_filter(self):
     @property
     def has_filtering(self):
         """Get whether or not the menu has any active filtering."""
-        return bool(self._active_filter and self._active_filter.filters)
+        return bool(self._active_filter and self._active_filter.filters or self._active_preset_filter_name)
```

Comment:
Even though these are equivalent, I would prefer to use parentheses here to make it more clear what the intention is
```suggestion
        return bool((self._active_filter and self._active_filter.filters) or self._active_preset_filter_name)
```

Reply:
yep good call, I agree that is just easier to read.